### ISSUE TITLE
[celery service] - Django Container Fails to Run After Generating Project with Latest cookiecutter-django

### ISSUE THREAD

**Issue by FilipeNas** [cookiecutter/cookiecutter-django#5242]
## What happened?

When starting a new project with celery option enabled, the following errors appear constantly:
```
Error: No nodes replied within time constraint
Celery workers not available.
```

And this error:
```
_rust_notify.WatchfilesRustInternalError: error in underlying watcher: IO error for operation on <path>: No such file or directory (os error 2)
```

## What should've happened instead?

No errors

## Additional details

Options used:
```
python_version = "3.12"
cloud_provider = 4
use_async = n
use_celery = y
```

**Comment by browniebroke** [comment]
A few follow up questions @FilipeNas:

- What OS are you using? Windows/Linux/macOS? If Windows are you on WSL or another environment?
- Are you using Docker?

**Comment by browniebroke** [comment]
This should be fixed in watchfiles 0.24, which we just upgraded to.

**Comment by FilipeNas** [comment]
sorry just saw now @browniebroke.
Windows 11, running docker dev in WSL.
